.gitignore

```diff
@@ -13,9 +13,8 @@ uploads/
 config/secrets.py
 package-lock.json
 
-# backup folder contains the issues.db backup file with issues dump
-# for the previous versions
-backups/*.db
+# data folder contains everything we are not interested
```

Comment:
Can we update this comment to something like `the data folder contains information that shouldn't live in version control` (or just fix the typo at the end and say `interested in`)

Reply:
@miketaylr Done! :) Thanks for the review.